controller.go

```diff
@@ -272,7 +272,9 @@ func (c *Controller) viewPath() string {
 
 // Redirect sends the redirection response to url with status code.
 func (c *Controller) Redirect(url string, code int) {
+	logAccess(c.Ctx, nil, code)
 	c.Ctx.Redirect(code, url)
+	panic(ErrAbort)
```

Comment:
Why add panic here?

Reply:
To stop the request and send a redirect header so it works the same as Abort() function. It rarely happens that the user would need to continue with the connection.